Pipfile

```diff
@@ -18,10 +18,10 @@ black = "*"
 build = "*"
 faker = "*"
 flake8 = "*"
-mcap = "==0.0.15"
-mcap-ros1-support = "==0.4.0"
-mcap-protobuf-support = "==0.0.8"
-mcap-ros2-support = "==0.1.0"
+mcap = ">=1.1.0"
```

Comment:
Didn't we have issues in the past with specifying floating versions? Or is the lockfile handling that now.

Reply:
I've changed this to `==1.1.0`, and updated the lockfile (running with python 3.9, as 3.10 removed [typing-extensions](https://pypi.org/project/typing-extensions/)).